src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball firstBall = lottery.getRandomBall();
+        Ball secondBall = lottery.getRandomBall();
+        Ball thirdBall = lottery.getRandomBall();
+        System.out.println("1st " + firstBall.toString());
+        System.out.println("2nd " + secondBall.toString());
+        System.out.println("3rd " + thirdBall.toString());
```

Comment:
let's use for loop here to create and print balls

Reply:
Is it a good practice to create const variable for count of balls?
Example:
`private static final int NUMBER_OF_BALLS = 3;`